### DISCUSSION TITLE
Enable PerformanceInsights on RDS restore from snapshot

### DISCUSSION THREAD

**Discussion by biohazd** [aws/aws-cli#7391] [Q&A] [answered]
Enable PerformanceInsights on rds restore from snapshot.

In the docs it does not say if this is enabled or not when restoring.

https://docs.aws.amazon.com/cli/latest/reference/rds/restore-db-instance-from-db-snapshot.html

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @biohazd, thanks for reaching out.  I did some testing, and it looks like RDS instances created through the `restore-db-instance-from-db-snapshot` method default to having performance insights **off**.  Please let me know if you have any other questions.

**Reply by biohazd** [reply]
Thanks.

is it possible to add a flag so it can be enabled when the database is restored.  It will save me having to restore and then enable it in a separate command.

**Reply by RyanFitzSimmonsAK** [reply]
If you'd like me to forward that feedback to the RDS team, I'd be happy to do so.  In general though, it's easier for your feedback to reach the right people through https://aws.amazon.com/contact-us/.  It's also easier for you to stay in the loop about your request using that process.
